### DISCUSSION TITLE
Easier development process?

### DISCUSSION THREAD

**Discussion by lordfolken** [Openvario/meta-openvario#211] [Q&A]
I currently build an image flash it to SD and thumble it into my openvario. This process is cumbersome and not very iteration friendly.

I wanted to know how you guys do it?

There seems to be a qemu emulation, but the required configuration file is missing:
```bash
/openvario/poky/scripts/runqemu openvario-image-openvario-57-lvds.ext4
runqemu - WARNING - /home/folken/openvario/poky/build/tmp/work/openvario_57_lvds-ovlinux-linux-gnueabi/openvario-image/1.0-r0/deploy-openvario-image-image-complete/openvario-image-openvario-57-lvds.qemuboot.conf doesn't exist
Traceback (most recent call last):
  File "/home/folken/openvario/poky/scripts/runqemu", line 1583, in main
    config.check_args()
  File "/home/folken/openvario/poky/scripts/runqemu", line 545, in check_args
    s = re.search('^DEPLOY_DIR_IMAGE="(.*)"', self.bitbake_e, re.M)
  File "/usr/lib/python3.9/re.py", line 201, in search
    return _compile(pattern, flags).search(string)
TypeError: expected string or bytes-like object
runqemu - INFO - Cleaning up
```
Potentially network boot would be an option? There seems to be a guide here: 
https://linux-sunxi.org/How_to_boot_the_A10_or_A20_over_the_network

**Comment by mihu-ov** [comment]
You can update the SD card from an USB drive, see  https://www.openvario.org/doku.php?id=projects:series_00:build:advanced_topics "Write new image from USB"
This is probably not what you are looking for but at least doesn´t require fiddling with the µSD card which is a pain. If you need to fiddle with the µSD card a short length of adhesive tape attached to it helps and avoids dropping the µSD inside the OV housing.
